### ISSUE TITLE
Create an app

### ISSUE THREAD

**Issue by i315315** [i315315/amos2022ws02-automotive-test-app#1]
## User story
1. As a User
2. I want to be able to build an APK of the test app that can be installed on an emulator or an Android Automotive test setup
3. So that I can start testing a new app

## Acceptance criteria
* The source code of an Android app is available.
* The app can be built to an APK.
* When the app is started, a (still empty) screen is displayed, from which the contexts can be entered later.
* The app is programmed in Kotlin (technically Java is also possible).

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
